specification/electronic-prescription-service-api.yaml

```diff
@@ -55,19 +55,31 @@ info:
     - [Personal Demographics Service - FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) - use this to obtain patient data
 
     ## API status and roadmap
-    The immediate EPS development priority is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
+    The immediate priority of EPS development is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
     
-    To suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
-
-    If you have any other queries, please [contact us](https://digital.nhs.uk/developer/help-and-support).
-
     ### Prescribing API
-    Prescribing flows for EPS FHIR API are now in a state of technical go-live awaiting our first of type integration.
-
-    The API is currently available for sandbox testing and integration testing, but not for production use.
-
+    There are three types of medication workflows
+    
+    #### Primary care medication workflow
+    In primary care medication workflow, the prescription is sent from a primary care service, such as a GP practise to a Pharmacy. For this workflow, the API is in [alpha](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status), meaning:
+    - it is available for testing in the integration environment, but not for production use
+    - we might make breaking changes, but only if we cannot avoid it, and we will give advance notice
+
+    #### Secondary care medication workflow
+    In a secondary care medication workflow, the prescription is sent from a secondary care service, such as an NHS hospital to a Pharmacy. For this workflow, the API is in [beta](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status), meaning:
+    - it is available in integration and production
+    - we might make breaking changes, but only if we cannot avoid it, and we will give advance notice
+    
+    #### Tertiary care medication workflow
+    This API does not support tertiary care medication workflow.
```

Comment:
_workflows_?

Reply:
Are there more than one workflow in Tertiary care?